pom.xml

```diff
@@ -234,6 +231,11 @@
                 <![CDATA[
                   {
                     "revapi" : {
+                      "semver": {
+                        "ignore": {
+                          "enabled": true
```

Comment:
if there is an incompatible change, it will fail the build and you can remedy it by running `mvn package revapi:update-versions` which will automatically change the version in the pom.

So in another words yes and yes ;)